util/pingpong.c

```diff
@@ -2167,7 +2167,7 @@ int main(int argc, char **argv)
 		return EXIT_FAILURE;
 	ct.hints->ep_attr->type = FI_EP_DGRAM;
 	ct.hints->caps = FI_MSG;
-	ct.hints->mode = FI_CONTEXT;
+	ct.hints->mode = FI_CONTEXT2;
```

Comment:
I agree that at the very least we need FI_CONTEXT | FI_CONTEXT2 here.

Reply:
Ok .. I'll make this change in one commit, then look into passing the correct structure based on fi_getinfo() in another commit.

Do I just close this PR and then make another with the fixed up commit?

Reply:
Just do a force push with the update and second commit.  That keeps this conversion associated with the changes. Thanks